### PR TITLE
PIKIP-15 Added profile settings and small styling fixes

### PR DESCRIPTION
**Pull request by rokaskasperavicius**

### CODE REVIEW

File: _sass/components/_components.Form.scss [hunk 1]
```diff
@@ -8,12 +8,16 @@
...
-
-    .field:nth-child(2n) {
-      padding-left: 30px;
+  &--create { 
+    .wrapper {
+      margin-top: 15px;
+      padding-bottom: 20px;
+      display: flex;
+      justify-content: center;;
+
```
Comment (ChristianMeiselsAsmussen) [inline _sass/components/_components.Form.scss:17]: double ;

File: _sass/components/_components.Section.scss [hunk 2]
```diff
@@ -14,49 +20,62 @@
...
-        bottom: -5px;
-        left: 0;
-        background-color: $primary;
-        opacity: 0;
-        height: 2px;
-        width: 100%;
-      }
+        position: relative;
+        color: gray;
+        cursor: pointer;
```
Comment (ChristianMeiselsAsmussen) [inline _sass/components/_components.Section.scss:39]: aren't you using color variables

File: _sass/components/_components.Section.scss [hunk 3]
```diff
@@ -14,49 +20,62 @@
...
-
-      &.active {
-        color: black;
-        
+        display: inline-block;
+        white-space: nowrap;
+        max-width: 250px;
+        text-overflow: ellipsis;
+        overflow: hidden;
+        flex-shrink: 0;
```
Comment (ChristianMeiselsAsmussen) [inline _sass/components/_components.Section.scss:48]: 250px was something I wrote - is it what you want?

**Review by ChristianMeiselsAsmussen — CHANGES_REQUESTED** [review]
minor things

File: _sass/components/_components.Section.scss (hunk 3 above)
Comment (rokaskasperavicius) [reply _sass/components/_components.Section.scss:48]: I don't think it matters that much in my opinion. The max characters the name's length can be 100. So, I think the 250px is long enough to display most of it

**Pull request merged**